models/migrations/migrations_test.go

```diff
@@ -204,7 +203,7 @@ func prepareTestEnv(t *testing.T, skip int, syncModels ...interface{}) (*xorm.En
 	deferFn := PrintCurrentTest(t, ourSkip)
 	assert.NoError(t, os.RemoveAll(setting.RepoRootPath))
 
-	assert.NoError(t, com.CopyDir(path.Join(filepath.Dir(setting.AppPath), "integrations/gitea-repositories-meta"),
+	assert.NoError(t, util.CopyDir(path.Join(filepath.Dir(setting.AppPath), "integrations/gitea-repositories-meta"),
```

Comment:
We can see that there was a PR:
* #14103

@6543 has tried to refactor the `com.Xxx` before, but these legacy functions still appears in new code ..... should we write the `com` into block list in lint rules?

Reply:
see https://gitea.com/gitea/gitea-vet/pulls/20

Reply:
I think we do not need it.

![image](https://user-images.githubusercontent.com/2114189/159210245-d49b6978-895f-48c1-a72f-fa55f2104811.png)


My PR is almost done.